src/mixins/createCachedRenderMixin.ts

```diff
@@ -121,9 +126,15 @@ const createCachedRenderMixin: CachedRenderFactory = createStateful
 					props[key] = cachedRender.listeners[key];
 				}
 				const classes: { [index: string]: boolean; } = {};
+				const widgetClasses: string[] = widgetClassesMap.get(cachedRender) || [];
```

Comment:
You could set this in the initializer (where you know it should be just `[]` as well as `.own` a handle to remove it).